bin/check-dns.rb

```diff
@@ -51,6 +51,12 @@ class DNS < Sensu::Plugin::Check::CLI
          short: '-s SERVER',
          long: '--server SERVER'
 
+  option :port,
+         description: 'Port to use for resolution',
+         short: '-p PORT',
+         long: '--port PORT',
+         proc: proc(&:to_i)
```

Comment:
This is unnecessary. Dnsruby defaults to standard TCP port 53 when the option isn't given.